packages/compiler/lib/decorators.ts

```diff
@@ -419,6 +432,85 @@ export function getMaxValue(program: Program, target: Type): number | undefined
   return program.stateMap(maxValuesKey).get(target);
 }
 
+// -- @minValueExclusive decorator ---------------------
+
+const minValueExclusiveKey = createStateSymbol("minValueExslusive");
+
+export function $minValueExclusive(
+  context: DecoratorContext,
+  target: Scalar | ModelProperty,
+  minValueExclusive: number
+) {
+  validateDecoratorUniqueOnNode(context, target, $minValueExclusive);
+  validateDecoratorNotOnNode(context, target, $minValue, $minValueExclusive);
```

Comment:
What happens if you have 
```
@maxValue(10)
scalar A extends float32;

@maxValueExclusive(10)
scalar A extends B;
```

Reply:
Good question! Right now, not an error. But I believe this should be an error. I will implement that.